Makefile

```diff
@@ -65,14 +66,18 @@ run: manifests generate fmt vet ## Run a controller from your host.
 
 ##@ Deployment
 
+KO_ARGS ?= --platform linux/amd64
+KAPP_ARGS ?= --yes=false
+KAPP_NS ?= kube-system
+
 install: manifests ## Install CRDs into the K8s cluster specified in ~/.kube/config.
-	$(KUSTOMIZE) build config/crd | $(KO) apply -f -
+	$(KAPP) deploy -a service-binding -n $(KAPP_NS) $(KAPP_ARGS) -f <($(KUSTOMIZE) build config/crd)
 
-uninstall: manifests ## Uninstall CRDs from the K8s cluster specified in ~/.kube/config.
-	$(KUSTOMIZE) build config/crd | $(KO) delete -f -
+uninstall: ## Uninstall CRDs from the K8s cluster specified in ~/.kube/config.
+	$(KAPP) delete -a service-binding -n $(KAPP_NS) $(KAPP_ARGS)
```

Comment:
shouldn't this line be:

```
$(KAPP) delete -a service-binding -n $(KAPP_NS) $(KAPP_ARGS) -f <($(KUSTOMIZE) build config/crd)
```

Reply:
`kapp delete` does not accept files. It will delete all of the resources associated with the app.